web/js/codeworld.js

```diff
@@ -1121,24 +1123,26 @@ function compile() {
     allowEscapeKey: false,
     allowEnterKey: false,
   }).then(() => {
-    window.cancelCompile();
+    stopRun();
   });
 
   sendHttp('POST', 'compile', data, (request) => {
     if (compileFinished) return;
-    sweetAlert.close();
```

Comment:
I think you will need to do this later if there are compile errors (see line 1181 where error messages are displayed when `success` is false`)

Reply:
Oh yeah, that's true, I missed that. Thanks!